### PR TITLE
Create base labels

### PR DESCRIPTION
**Pull request by hellcp-work**

### CODE REVIEW

File: src/api/db/schema.rb [hunk 1]
```diff
@@ -1176,7 +1196,7 @@
     t.index ["state"], name: "index_users_on_state"
   end
 
-  create_table "versions", charset: "utf8mb4", collation: "utf8mb4_general_ci", force: :cascade do |t|
+  create_table "versions", charset: "utf8mb4", force: :cascade do |t|
```
Comment (hennevogel) [inline src/api/db/schema.rb:1199]: This looks unrelated...

Comment (eduardoj) [reply src/api/db/schema.rb]: Ops. Fixed. Thanks!

**Review by hennevogel — APPROVED** [review]
